guides/common/modules/proc_configuring-satellite-deployment-with-an-external-dhcp-server.adoc

```diff
@@ -79,13 +79,5 @@ bash-4.2$ exit
 --enable-foreman-proxy-plugin-dhcp-remote-isc \
 --foreman-proxy-dhcp-server=_DHCP_Server_FQDN_
 ----
-. Restart the `foreman-proxy` service:
-+
-[options="nowrap"]
-----
-# systemctl restart foreman-proxy
-----
 [cols="40%,60%",options="header"]
```

Comment:
I don't understand this table header here. Does it have any specific purpose?

Reply:
Excellent question. I'll just admit that I only spotted this whole redundancy while reading through it because of NFS fixes in https://github.com/theforeman/foreman-documentation/pull/1827, which in itself was already scope creep. Doesn't look like it's doing anything so I'll drop it (and fix the typo in my commit message while I'm at it).